From ad1849206493d56aa1547e5d491249f60219dd58 Mon Sep 17 00:00:00 2001
Message-ID: <ad1849206493d56aa1547e5d491249f60219dd58.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:50:47 +0100
Subject: [PATCH v3 17/66] gnu: go-github-com-jcmturner-dnsutils-v2: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-jcmturner-dnsutils-v2): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I65f87f8e8cd7970a15d3b1e09c5ef4202f0542db
---
 gnu/packages/golang-web.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 27 ---------------------------
 2 files changed, 28 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 580877f8fe..9b2bcd7cbd 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -286,6 +286,34 @@ (define-public go-github-com-gorilla-websocket
 protocol.")
     (license license:bsd-2)))
 
+(define-public go-github-com-jcmturner-dnsutils-v2
+  (package
+    (name "go-github-com-jcmturner-dnsutils-v2")
+    (version "2.0.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/jcmturner/dnsutils")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "116zbgvfj88vv93fnapmmgyd5g8kzy774cdyzsnnzyzng92j61c9"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/jcmturner/dnsutils/v2"))
+    (propagated-inputs
+     (list go-github-com-stretchr-testify))
+    (home-page "https://github.com/jcmturner/dnsutils")
+    (synopsis "Go library with DNS utils")
+    (description
+     "The dnsutils package provides a Go function to return a map of Service
+Records (SRV) in the order they should be used for a given service, protocol
+and name.  The order is determined by the records' priority and randomized
+selection based on their relative weighting.  This package is useful for
+network applications that require accessing services using SRV records.")
+    (license license:asl2.0)))
+
 (define-public go-github-com-jcmturner-gokrb5-v8
   (package
     (name "go-github-com-jcmturner-gokrb5-v8")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index b0ae0da3e3..d0cde62f61 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1363,33 +1363,6 @@ (define-public go-github-com-jcmturner-aescts-v2
 encryption and decryption methods.")
     (license license:asl2.0)))
 
-(define-public go-github-com-jcmturner-dnsutils-v2
-  (package
-    (name "go-github-com-jcmturner-dnsutils-v2")
-    (version "2.0.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/jcmturner/dnsutils")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "116zbgvfj88vv93fnapmmgyd5g8kzy774cdyzsnnzyzng92j61c9"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/jcmturner/dnsutils/v2"))
-    (propagated-inputs (list go-github-com-stretchr-testify))
-    (home-page "https://github.com/jcmturner/dnsutils")
-    (synopsis "Go library with DNS utils")
-    (description
-     "The dnsutils package provides a Go function to return a map of Service
-Records (SRV) in the order they should be used for a given service, protocol
-and name.  The order is determined by the records' priority and randomized
-selection based on their relative weighting.  This package is useful for
-network applications that require accessing services using SRV records.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-jcmturner-goidentity-v6
   (package
     (name "go-github-com-jcmturner-goidentity-v6")
-- 
2.41.0

